src/recensio/plone/behaviors/base.py

```diff
@@ -83,6 +83,7 @@ class IBase(model.Schema):
         title=_("Generated Pdf"),
         required=False,
     )
+    searchable("generatedPdf")
```

Comment:
```suggestion
```

This is generated from the field `review` which is already searchable.

Reply:
Agreed